ui/app/templates/components/clients/current.hbs

```diff
@@ -0,0 +1,70 @@
+<div class="box is-sideless is-fullwidth is-marginless is-bottomless">
+  <p class="has-bottom-margin-s">
+    The below data is for the current month starting from the first day. For historical data, see the monthly history tab.
+  </p>
+  {{#if (eq @model.config.enabled "Off")}}
+    <EmptyState
+      @title="Tracking is disabled"
+      @message="Tracking is disabled and data is not being collected. To turn it on edit the configuration."
+    >
+      {{#if @model.config.configPath.canUpdate}}
+        <LinkTo @route="vault.cluster.clients.edit">
+          Go to configuration
+        </LinkTo>
+      {{/if}}
+    </EmptyState>
+  {{/if}}
+
+  {{#if @isLoading}}
+    <LayoutLoading />
+  {{else}}
+    <div class="card has-bottom-margin-m">
```

Comment:
I think we might be able to use the <Clients::SingleMonth> template here (can be added in future PR once components are wired up correctly)

Reply:
Or partial month, however it's referred to in the API. but agree